### ISSUE TITLE
Empty tuple printed at startup on Python 2

### ISSUE THREAD

**Issue by jakirkham** [jupyter/jupyter_console#64]
When starting up the console on Python 2, there is an empty tuple. This is not present on Python 3. Likely due to some missing `from __future__ import print_function` and a call like `print()`.

```
$ jupyter console 
Jupyter Console 4.1.0
()

In [1]:
```

Details from conda.

```
$ conda list
# packages in environment at /zopt/conda2/envs/python2:
#
appnope                   0.1.0                    py27_0    defaults
backports-abc             0.4                       <pip>
backports.ssl-match-hostname 3.4.0.2                   <pip>
backports_abc             0.4                      py27_0    defaults
decorator                 4.0.9                    py27_0    defaults
ipykernel                 4.3.1                    py27_0    defaults
ipython                   4.1.2                    py27_0    defaults
ipython-genutils          0.1.0                     <pip>
ipython_genutils          0.1.0                    py27_0    defaults
jupyter-client            4.1.1                     <pip>
jupyter-console           4.1.0                     <pip>
jupyter-core              4.0.6                     <pip>
jupyter_client            4.1.1                    py27_0    defaults
jupyter_console           4.1.0                    py27_0    defaults
jupyter_core              4.0.6                    py27_0    defaults
openssl                   1.0.2g                        0    defaults
path.py                   8.1.2                    py27_1    defaults
pexpect                   3.3                      py27_0    defaults
pickleshare               0.5                      py27_0    defaults
pip                       8.0.3                    py27_0    defaults
python                    2.7.11                        0    defaults
python.app                1.2                      py27_4    defaults
pyzmq                     15.2.0                   py27_0    defaults
readline                  6.2                           2    defaults
setuptools                20.1.1                   py27_0    defaults
simplegeneric             0.8.1                    py27_0    defaults
singledispatch            3.4.0.3                  py27_0    defaults
six                       1.10.0                   py27_0    defaults
sqlite                    3.9.2                         0    defaults
ssl_match_hostname        3.4.0.2                  py27_0    defaults
tk                        8.5.18                        0    defaults
tornado                   4.3                      py27_0    defaults
traitlets                 4.1.0                    py27_0    defaults
wheel                     0.29.0                   py27_0    defaults
zlib                      1.2.8                         0    defaults
```

**Comment by takluyver** [comment]
Should have been fixed by #60. Update to 4.1.1.

**Comment by jakirkham** [comment]
Thanks for the pointer. That does seem to fix it. Though 4.1.1 isn't available in conda yet.
